### DISCUSSION TITLE
作品広告

### DISCUSSION THREAD

**Discussion by nakatsuka-k** [aipictors/aipictors#53] [アイデア - Ideas]
忙海AI さんより
＞課金ユーザーの作品をピックアップとして表示させるようにしたり、
＞おすすめ作品に登場し易くしたら課金ユーザーの方のモチベーションに繋がる

広告枠をつくって、だれでも作品を紹介できるようにするといいと思われる

**Comment by mikageymir** [comment]
案：
スタンダードで1作品、プレミアムで2作品えらび、全年齢・R15作品の広告枠に選んだ作品を出せます。（掲出はランダム）

<img width="418" alt="image" src="https://github.com/aipictors/aipictors/assets/22807407/713efd29-c40e-458c-bb1a-f69984320f37">
